### ISSUE TITLE
You cannot call deon via `-m`

### ISSUE THREAD

**Issue by koaning** [drivendataorg/deon#112]
In python it can be safer to run command line apps via; 

```
(venv) > python -m jupyter lab 
```

This way, even if `jupyter` is not installed in this virtual env I'll get an error instead of it grabbing the jupyter from the global env. 

Unfortunately deon does not support this. 

```
> python -m deon
/Users/vincent/Development/calmcode.io/venv/bin/python: No module named deon.__main__; 'deon' is a package and cannot be directly executed
```

I can pick this issue up if folks agree that adding `__main__.py` makes sense. I'd love to contribute to this project.

**Comment by jayqi** [comment]
Cool! TIL about `__main__.py`. That sounds like a great thing to add. We welcome the contribution.

**Comment by koaning** [comment]
Sweet, you can find the PR [here](https://github.com/drivendataorg/deon/pull/113). 

Also, I wrote a small [tutorial](https://calmcode.io/deon/introduction.html) for this package on my blog. This is a cool project.

**Comment by jayqi** [comment]
@koaning We just released v0.2.2 with your change. Thanks for the contribution!

Closed by #113.
